README.md

```diff
@@ -89,3 +89,17 @@ familiar aliases like:
 * gocs = A cs
 * gopeers = A peers
 * goshare = A share
+
+## Git
+
+For people that have some mental problems and need to commit and push every
+small stable step they give there is the **gitcp** command that, given a message,
+will commit all changes with that message and already push it too.
+
+Example:
+
+```
+gitcp 'my awesome message'
```

Comment:
`gitsh` must be something related to `git` and `shell`, and in the same way, `gitcp` must be something related to `git` and `cp`. I know this should not be a global rule, but people immortalized the `sh`, `cp`, `cat`, and so on. I think this name is misleading.

As we always use git as VCS, maybe we could omit the `git` part of the command name and put the tools inside a `git` folder.
Some names that pops out with this new convention are:

`git/push`: git push origin <current branch>
`git/cpush`: git commit && git push
`git/pull`: git pull origin <current branch>

Then you can add the `git` folder to PATH also and use just `push`, `cpush`, etc.

When coding, the term `push` is always recognized as `git push`, then makes sense it being just it inside the acme but when used outside the editor requires invoking by `git/push` to add the proper context.

Reply:
I did not liked the name gitcp either because really remembers copy =P, but giving names is hard and sometimes I get lazy =(. Im prone to simply renaming it to "cpush". We are already on a very narrow namespace (the acme edit dir), probably we are not going to have another "cpush" there...I think =P.